src/models/user.js

```diff
@@ -0,0 +1,26 @@
+import mongoose from "mongoose";
```

Comment:
Why can't you use userValidation file? Try to separate the model with validation for the readability of the codes